package.json

```diff
@@ -2,24 +2,37 @@
   "name": "@pxblue/highcharts",
   "version": "1.1.1",
   "description": "Base configurations for assorted HighCharts graph types for PX Blue.",
-  "main": "./dist/index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
   "scripts": {
-    "test:build": "./buildTest.sh",
+    "build": "babel src --presets=@babel/preset-env --out-dir dist",
     "test": "jest",
-    "build": "babel src --presets=@babel/preset-env --out-dir dist"
+    "test:build": "bash ./buildTest.sh",
+    "build:react": "cd react-demo && yarn build && cd ..",
```

Comment:
we don't need to script out building the demos...we just run them locally..see the updates I made in chartjs